corehq/apps/data_cleaning/tests/test_session.py

```diff
@@ -192,6 +192,31 @@ def test_reorder_filters(self):
             ['name', 'watered_on', 'num_leaves', 'height_cm', 'pot_type']
         )
 
+    def test_reorder_wrong_number_of_column_ids_raises_error(self):
+        session = BulkEditSession.new_case_session(self.django_user, self.domain_name, self.case_type)
+        columns = session.columns.all()
+        new_order = [columns[1].column_id, columns[2].column_id]
+        with self.assertRaises(ValueError):
+            session.update_column_order(new_order)
+
+    def test_update_column_order(self):
+        session = BulkEditSession.new_case_session(self.django_user, self.domain_name, self.case_type)
+        columns = session.columns.all()
+        new_order = [
+            columns[1].column_id,
+            columns[0].column_id,
+            columns[2].column_id,
+            columns[4].column_id,
+            columns[5].column_id,
+            columns[3].column_id,
+        ]
+        session.update_column_order(new_order)
+        reordered_prop_ids = [c.prop_id for c in session.columns.all()]
+        self.assertEqual(
+            reordered_prop_ids,
+            ['owner_name', 'name', 'date_opened', 'last_modified', '@status', 'opened_by_username']
```

Comment:
Is there something like `columns[1].name` to refer instead of hard coding it as `'owner_name'` ?

Reply:
i think it's fine to reference the names for now. i'll update and revisit all of these tests later.